### DISCUSSION TITLE
为什么设置的速度下限不管用呢？

### DISCUSSION THREAD

**Discussion by Idealisten** [XIU2/CloudflareSpeedTest#434] [提问]
明明设置了5MB的速度下限，结果返回了4个速度不符合的，好奇怪。。。
![](https://picture.runtofree.link/picture/2023/09/eee2c4de5eade23dada7e4db3232ef10.png)

**Comment by XIU2** [comment]
当没有找到 1 个满足条件的 IP 时，就会直接把所有测速结果显示出来，这样你才能知道 下载速度下限 设置的过高了，下次就可以适量降低 下载速度下限。
